raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -1086,6 +1086,16 @@ contract TokenNetwork is Utils {
         }
     }
 
+    /// @dev Get the hash of the channel identifier and the participant
+    /// addresses (whose orderings matter). The hash might be useful for
+    /// looking up an UnlockData in
```

Comment:
`looking up an UnlockData in` -> `looking up the appropriate UnlockData for the participant`

Reply:
will change into `... useful for the partner to look up the appropriate UnlockData to claim.`